deploy/001-comptroller.ts

```diff
@@ -6,20 +9,43 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const { deploy } = deployments;
 
   const { deployer } = await getNamedAccounts();
+  const deployerSigner = await hre.ethers.getSigner(deployer);
 
-  await deploy("AccessControlManager", {
+  const accessControlManagerDeployment = await deploy("AccessControlManager", {
     from: deployer,
     args: [],
     log: true,
     autoMine: true,
   });
```

Comment:
I think we should also cosider ACM for live network, If `hre.network==live` await ethers.getContract("AccessControlManager"